packages/client/src/components/ClientProvider/ClientProvider.tsx

```diff
@@ -1,17 +1,15 @@
-import { authExchange } from "@urql/exchange-auth";
+import React, { useMemo } from "react";
+import { AuthUtilities, authExchange } from "@urql/exchange-auth";
```

Comment:
`AuthUtilities` is defined but never used

```suggestion
import { authExchange } from "@urql/exchange-auth";
```